index.js

```diff
@@ -238,6 +238,15 @@ io.on('connection', function(socket){
                                     }else if(cell.build === builds.WAREHOUSE){
                                         cell.owner = userData.security; // Musíme přepsat již nyní, aby se správně propočetla nová maxima skladu
                                         utils.updatePlayerMaxResources(oldOwner);
+                                    }else if(cell.build === builds.ROCK){
+                                        if(userData.energy < 10) return; // Obsadenie skaly stojí 10 energie
+                                            energyCost = 10;
+
+                                            db.world.cellUpdate(x, y, userData.security, cell.build);
+                                            io.emit('cell', x, y, userData.username, userData.color, cell.build);
+
+                                            userData.energy -= 10;
+                                            socket.emit('info', { energy: userData.energy });
```

Comment:
Celá tahle část je tu úplně zbytečně. Proč to tu je?

Reply:
Teraz som si to uvedomil, bude upravené.

Reply:
Táto časť tam je na obsadenie skaly ktoré stojí 10 energie.

Reply:
O řádek nad tím nastavuješ už `energyCost` na 10, to se odečte dole, kde se to ukládá. Zbytečně tu rozepisuješ část kódu, která už je řešena níže.